pkg/server/test/list_objects.go

```diff
@@ -29,6 +29,119 @@ func (x *mockStreamServer) Send(m *openfgapb.StreamedListObjectsResponse) error
 	return nil
 }
 
+// mockSlowDataStorage is a proxy to the actual ds except the Reads are slow time by the sleepTime
+// This allows simulating list objection condition that times out
+type mockSlowDataStorage struct {
```

Comment:
Please put mocked storage interfaces in the `pkg/storage/mocks` package.

Reply:
Done